src/Interface/ResultTypeInferenceInterface.cpp

```diff
@@ -0,0 +1,20 @@
+//===------------ ResultTypeInferenceInterface.cpp --------------===//
+//===--------- Infer Data Type for Results Interface Definition --------===//
+//
+// Copyright 2020 The IBM Research Authors.
+//
+// =============================================================================
+//
+// This file contains the declaration of the data type reference  interface.
```

Comment:
declaration -> implementation.
reference -> inference.

Please remove the whitespace before interface.